keras/engine/training.py

```diff
@@ -1427,7 +1523,9 @@ class indices (integers) to
                               verbose=verbose, callbacks=callbacks,
                               val_f=val_f, val_ins=val_ins, shuffle=shuffle,
                               callback_metrics=callback_metrics,
-                              initial_epoch=initial_epoch)
+                              initial_epoch=initial_epoch,
+                              steps_per_epoch=steps_per_epoch,
+                              validation_steps=validation_steps)
 
     def evaluate(self, x, y, batch_size=32, verbose=1, sample_weight=None):
```

Comment:
Ah the most recent post didn't load, also I had changed my mind as well and added evaluate since it was only a couple lines. I'll look at predict too.